src/app/books/books.component.scss

```diff
@@ -0,0 +1,10 @@
+.book-list {
+  margin: 2rem 1rem;
+  display: flex;
```

Comment:
You can use FlexLayoutModule instead of 
display: flex;
flex-direction: column;
gap: 1rem;